lib/phone_shop/widgets/components/shop_gallery.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+class ShopGallery extends StatelessWidget {
+  final List<String> images;
+  final EdgeInsetsGeometry padding;
+
+  const ShopGallery({super.key, required this.images, required this.padding});
+
+  @override
+  Widget build(BuildContext context) {
+    Widget productListView;
+
+    productListView = ListView.builder(
+      scrollDirection: Axis.horizontal,
+      itemCount: images.length,
+      itemBuilder: (context, index) {
+        return Padding(
+          padding: padding,
+          child: _ImageItem(image: images[index]),
+        );
+      },
+    );
+    return SizedBox(
+      height: 328,
+      child: productListView,
+    );
+  }
+}
+
+class _ImageItem extends StatelessWidget {
+  final String image;
+
+  const _ImageItem({
+    required this.image,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ClipRRect(
+      borderRadius: BorderRadius.circular(20),
+      child: Image.network(
```

Comment:
Image should have height and width, try why whit this image [https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fitzone.com.vn%2Fwp-content%2Fuploads%2F2022%2F10%2Fiphone-15-1664543780049419585259-1664610757213-166461075872649161420.jpeg&f=1&nofb=1&ipt=ea176cd11c9816e99ab704b0da377bfd04ef5ac5c6d2c8d541098a6936400e2c&ipo=images](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fitzone.com.vn%2Fwp-content%2Fuploads%2F2022%2F10%2Fiphone-15-1664543780049419585259-1664610757213-166461075872649161420.jpeg&f=1&nofb=1&ipt=ea176cd11c9816e99ab704b0da377bfd04ef5ac5c6d2c8d541098a6936400e2c&ipo=images)